themes/default/static/programs/aws-s3-bucket-resources-csharp/aws-s3-bucket-resources-csharp.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
```

Similar to your other PR, we should make this `net6.0`:

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```